{"name": "test_add_caculator_error[小数]", "status": "passed", "description": "这是假发计算器，测试加法有效各种情况，结果是否正确", "steps": [{"name": "第一步：调用加法的方法", "status": "passed", "start": 1599726569808, "stop": 1599726569808}, {"name": "第二部：断言加法运算结果与预期是否一致", "status": "passed", "start": 1599726569808, "stop": 1599726569808}], "parameters": [{"name": "num1", "value": "6"}, {"name": "num2", "value": "-2"}, {"name": "result", "value": "4"}], "start": 1599726569808, "stop": 1599726569808, "uuid": "12a2dbf8-0f49-4beb-a743-853e92a2b470", "historyId": "e50da3f3cbeb84e147a4c9ce820747b6", "testCaseId": "5e3d2d6f942a082c51d923142be5b120", "fullName": "test_pytest_allure#test_add_caculator_error", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "加法的计算器测试"}, {"name": "story", "value": "正确的测试用例"}, {"name": "feature", "value": "www.baidu.com"}, {"name": "suite", "value": "test_pytest_allure"}, {"name": "host", "value": "DESKTOP-TBPR3U9"}, {"name": "thread", "value": "11916-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest_allure"}], "links": [{"type": "issue", "url": "www.baidu.com", "name": "曾经的bug"}]}